app/views/questions/show.html.erb

```diff
@@ -19,12 +19,17 @@
   <div class="prettyform answers" style="display:none">
     <div class="box center">
       <h3>Answer:</h3>
+      <div class="net-votes"><p id="question-<%= @question.id %>-net-votes"><%= @question.plusminus %></p><br></div>
```

Comment:
I would put `<p id="question-<%...></p><br>` on a separate line just for legibility and to match the convention throughout the rest of the code. Otherwise, it looks great!